docs/source/conf.py

```diff
@@ -10,14 +10,18 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
-# import os
+import os
 # import sys
 # sys.path.insert(0, os.path.abspath('.'))
 
 
 # -- Project information -----------------------------------------------------
+repo_slug = os.environ.get("TRAVIS_REPO_SLUG") 
+if repo_slug is not None:
+    project = repo_slug
+else:
+    project = os.environ.get("PROJECT_NAME", "Set 'PROEJECT_NAME'")
```

Comment:
Hmm, what about maybe:
```suggestion
    project = os.environ.get("PROJECT_NAME", os.path.split(os.path.abspath('../../'))[-1])
```

I think that would default to the directory name where it's cloned to.